app/inputs/select_with_modal_help_input.rb

```diff
@@ -31,6 +31,13 @@ def build_field(value, _index)
       @rendered_first_element = true
 
       html_options.merge!(options.slice(:include_blank))
-      template.select_tag(attribute_name, template.options_for_select(select_options, value), html_options)
+      render_options = select_options
+      if value.present? && render_options.find { |opt| opt[1] == value }.nil?
+        force_option = RightsService.select_inactive_options.find { |opt| opt[1] == value } || [value, value]
```

Comment:
It doesn't belong in the Rights Service, because that's concerned only with what rights are currently active or inactive in general, not with which are associated with works. And it's not exactly view logic (occurring in the view itself), because this class is a view helper already. But, the extra logic there could definitely be more cleanly handled in another private method on that class, like so:
```
    def add_force_option_data_to_html_options(select_options, value, html_options)
      if value.present? && select_options.find { |opt| opt[1] == value }.nil?
        force_option = RightsService.select_inactive_options.find { |opt| opt[1] == value } || [value, value]
        html_options[:class] << "force-select"
        html_options[:'data-force-label'] = force_option[0]
        html_options[:'data-force-value'] = force_option[1]
      end
      html_options
    end
```

and then in the build_field method you get this:

```
      html_options.merge!(options.slice(:include_blank))
      html_options = add_force_option_data_to_html_options(select_options, value, html_options)
      template.select_tag(attribute_name, template.options_for_select(select_options, value), html_options)
```

I like this, what do you think?